packages/accounts/src/kernel-zerodev/e2e-tests/mocks/mock-signer.ts

```diff
@@ -0,0 +1,13 @@
+import type { Address, Hex, SmartAccountSigner } from "@alchemy/aa-core";
```

Comment:
mocked like how zerodev did, in order to avoid using .env variables like owner nemonic for unit tests so devs can run the unit tests locally. I couldn't use a dummy nemonic like in local-account because the checksum invalid with the ethers library.